app/models/vm_scan.rb

```diff
@@ -600,6 +556,32 @@ def abort_retry(*args)
 
   private
 
+  def create_snapshot(vm)
```

Comment:
@chessbyte Line 578 will return true.  If this is what @roliveri was referring to, I can add a `return true` after 578 to clear up confusion.